apps/admin/frontend/src/screens/tally/confirm_remove_cvrs_modal.tsx

```diff
@@ -45,18 +44,15 @@ export function ConfirmRemoveCvrsModal({
     onClose();
   }
 
-  if (isTestMode && hasManualData) {
+  if (hasManualData) {
     const anyMutationIsLoading =
       clearCastVoteRecordFilesMutation.isLoading ||
       deleteAllManualResultsMutation.isLoading;
     return (
       <Modal
-        title="Remove All Results?"
+        title="Remove All CVRs"
         content={
-          <P>
-            To reset this machine after testing, you must remove all CVRs and
-            manual tallies.
-          </P>
+          <P>Tallies will be removed from reports and permanently deleted.</P>
```

Comment:
<img width="543" alt="Screen Shot 2024-10-09 at 11 21 02 AM" src="https://github.com/user-attachments/assets/1dacb10f-4ad7-4b40-b7e2-df56cd0b4db2">
<img width="546" alt="Screen Shot 2024-10-09 at 11 20 07 AM" src="https://github.com/user-attachments/assets/bb8889fa-5d7c-4a4e-b800-d8c6e93ba13e">

Reply:
@mattroe I actually deviated from what we decided. I thought it was confusing to jump from a button that says "Remove CVRs" to "Remove All Tallies," so I just made both "Remove All CVRs".

Reply:
@adghayes and I discussed and decided to move the warning to also remove tallies to a second step after removing CVRs